orttraining/orttraining/test/gradient/gradient_ops_test.cc

```diff
@@ -50,18 +50,44 @@ static bool IsErrorWithinTolerance(float error, float tolerance) {
 static void RunReductionTests(const OpDef& op_def, bool axes_as_input = false,
                               bool check_not_have_shape_inferencing = false) {
   std::vector<std::vector<int64_t>> x_shapes = {
-      {4, 3, 2}, {4, 3, 2}, {4, 3, 2}, {4, 3, 2}, {4, 3, 2}, {4, 3, 2}, {4, 3, 2}, {4, 3, 2},
+      {4, 3, 2},
+      {4, 3, 2},
```

Comment:
nit: we'd better avoid making this kind of change to make it easier for us when merge back to master branch.

Reply:
we'll be cleaning up anyway at merge time, let's handle it then? will make sure to not make such changes further